client/components/tasks-list/index.js

```diff
@@ -48,24 +49,30 @@ export default class TasksList extends Component {
         </h3>
         <table>
           <tbody>
-            {serviceName && 
-              <tr>
-                <th>Service</th>
-                <td><Link to={serviceUrl}>{serviceName}</Link></td>
-              </tr>
-            }
-            <tr>
-              <th>Container Instance</th>
-              <td><Link to={containerInstanceUrl}>{containerInstanceId}</Link></td>
+          {serviceName &&
+          <tr>
+            <th>Service</th>
+            <td><Link to={serviceUrl}>{serviceName}</Link></td>
+          </tr>
+          }
+          <tr>
+            <th>Container Instance</th>
+            <td><Link to={containerInstanceUrl}>{containerInstanceId}</Link></td>
+          </tr>
+          <tr>
+            <th>Task ID</th>
+            <td><a href={taskUrl} target="_blank">{taskArnId}</a></td>
+          </tr>
+          {task.containers[0].networkBindings.map(({protocol, hostPort, containerPort}, index) => (
```

Comment:
If `task.containers` is a list, do you need to map over each container before mapping over its network bindings to make sure all containers are accounted for? Or is this list effectively _always_ a list of 1 item and this is fine?

Reply:
This is always a list of one, because a task (the instance of a task definition) runs on a container.